resource/environment/integration.js

```diff
@@ -201,7 +204,20 @@ function configure( envelope ) {
 
 function upgrade( slack, envelope ) {
 	var image = envelope.data.image;
-	return environment.getAll().then( onEnvironments.bind( null, image, slack ), onReadError );
+	var parsedPhase1 = image.split( ":" );
```

Comment:
since the parsing is only needed for the `dockerhub.checkExistance` call it should be moved into that function instead of doing it at this level